services/QuillLMS/engines/evidence/lib/evidence/check/base.rb

```diff
@@ -24,7 +26,7 @@ def self.run(entry, prompt, previous_feedback, session_uid = nil)
         context = { entry:, prompt_id: prompt&.id, prompt_text: prompt&.text }
         Evidence.error_notifier.report(e, context)
 
```

Comment:
This was an existing bug that I'm fixing. This is a class method, so assigning to @error wasn't assigning it to the object.